lib/pbench/server/api/__init__.py

```diff
@@ -10,7 +10,6 @@
 from flask import Flask
 from flask_cors import CORS
 from flask_restful import Api
-
 from pbench.common.exceptions import BadConfig, ConfigFileNotSpecified
 from pbench.common.logger import get_pbench_logger
 from pbench.server import PbenchServerConfig
```

Comment:
I think our convention is to separate the `pbench` imports from the other third-party imports by a blank line (so, you should put this one back...).

Reply:
No, idea how it got removed